### DISCUSSION TITLE
Issues creating a html m3u playlist player

### DISCUSSION THREAD

**Discussion by zeus1921** [videojs/video.js#8377] [Q&A]
Hey there.

I'm almost finishing my project, its a html m3u playlist player and you can see it [here](https://codepen.io/francisco-ribeiro-the-looper/pen/KKrBBXW)

Im facing some problems, some channels won't open, perhaps its on a diferent encoder or format? so i need to make the code to detect the output and open it correctly, how?

Can someone help me?

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! 👋

If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).

**Comment by mister-ben** [comment]
The codpen has video.js's CSS but a different player, but you'd have the same problems with the individual streams. Loading from http on an HTTPS website, or without cors headers on the streams permitting access won't work.
